recipes/perl-uri-query/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "perl-uri-query" %}
+{% set version = "0.16" %}
+{% set sha256 = "b4e62de79b468dcd7ee835e4dfd0035c83faf92e6c44b79bcdd9a50287fb8c18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/G/GA/GAVINC/URI-Query-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    weak:
+      - {{ name }} ={{ version }}
+
+requirements:
+  build:
+        - make
```

Comment:
Please fix the indentation here.

Reply:
Done! Thanks @xileF1337 (Sorry for the delay, back from holidays...)